src/pages/landingPage/explore/Explore.view.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import style from "./explore.module.css";
+
+const ExploreView = ({ concepts }) => {
+  console.log(concepts);
+
+  const conceptList = concepts.map((concept) => {
+    return (
+      <div className={style.card}>
+        <h3 className={style.main_text} key={concept.category}>
```

Comment:
pls make sure you maintain using `_` for CSS and communicate this to Abbi and Adrian. So we can keep this standard